va/va.go

```diff
@@ -738,6 +832,12 @@ func (va *ValidationAuthorityImpl) PerformValidation(ctx context.Context, domain
 	}
 	vStart := va.clk.Now()
 
+	var remoteError chan *probs.ProblemDetails
```

Comment:
I'd rather not as this requires that we do the `p, ok := err.(*probs.ProblemDetails); !ok || p != nil` dance on both sides of the channel and an extra cast so that we can put the `probs.ProblemDetails` into the `challenge.Error` field. I think this is one of the very few cases where using an actual `probs.ProblemDetails` actually makes sense.